### PR TITLE
docs(mkdocs): add config to use emoji

### PR DESCRIPTION
**Pull request by satoshi-ota**
### Before (can't use emoji)

image

### After

image

## Tests performed

https://autowarefoundation.github.io/autoware.universe/pr-3613/

## Effects on system behavior

Nothing.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: mkdocs.yaml [hunk 1]
```diff
@@ -80,3 +80,6 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
   - toc:
       permalink: "#"
+
```
Comment (kenji-miyake) [inline mkdocs.yaml:83]: [style]
- No unnecessary new line.
- Sort alphabetically.

```suggestion
```

Comment (kenji-miyake) [reply mkdocs.yaml:83]: FYI: Maybe adding sync settings like this is useful.
https://github.com/autowarefoundation/open-ad-kit-docs/blob/0c1c70625fb89d144a5e2d5df52d0cdccd6e18c4/.github/sync-files.yaml#L37-L41
https://github.com/autowarefoundation/autoware-documentation/blob/fa986159d4fd83d81f232f5ec1e657e5685f495f/mkdocs.yaml#L67-L69

Comment (satoshi-ota) [reply mkdocs.yaml:83]: I added sync in https://github.com/autowarefoundation/autoware.universe/pull/3620, and https://github.com/autowarefoundation/autoware.universe/pull/3619 is generated.

Would my these changes be as you intended?

Comment (kenji-miyake) [reply mkdocs.yaml:83]: Yes, exactly. Thank you!

Comment (satoshi-ota) [reply mkdocs.yaml:83]: Thanks for your comments!

**Comment by satoshi-ota** [comment]
I added sync in https://github.com/autowarefoundation/autoware.universe/pull/3620, and https://github.com/autowarefoundation/autoware.universe/pull/3619 is generated.
This PR is no longer needed. Close.

**Pull request closed without merging**